docs/starters.yml

```diff
@@ -1420,3 +1420,21 @@
     - Prettier integrated into ESLint
     - Jest for component testing
     - A few example components and pages with stories and simple site structure
+- url: https://vigilant-leakey-a4f8cd.netlify.com/
+  repo: https://github.com/BoyWithSilverWings/gatsby-blog-starter
+  description: Minimal Blog Starter Template with Styled Components.
+  tags:
+    - MDX
+    - Markdown
+    - Styled Components
+    - Styling:CSS-in-JS
+    - Blog
+  features:
+    - Gatsby MDX for JSX in Markdown loading, parsing, and rendering of pages
```

Comment:
As above: I don't see you using MDX so you should probably correct this line

Reply:
@LekoArts  Done with above changes.